tests/js/HappinessLintBearTest.py

```diff
@@ -1,6 +1,10 @@
+from queue import Queue
 from bears.js.HappinessLintBear import HappinessLintBear
 from coalib.testing.LocalBearTestHelper import verify_local_bear
-
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.BearTestHelper import generate_skip_decorator
 
```

Comment:
please add here an empty line so it's like before^^

Reply:
one more line?